### PR TITLE
DATA-2190 Insert predicted values to dynamodb

### PR DESCRIPTION
**Pull request by PatrikSakac**

### CODE REVIEW

File: mlflow/pyfunc/scoring_server.py [hunk 1]
```diff
@@ -177,6 +197,20 @@ def transformation():  # pylint: disable=unused-variable
...
             raw_predictions = model.predict(data)
+            for i in range(len(data)):
+                log_mlflow_predictions.put_item(
+                    Item={
+                        'lead_id': int(data['lead_id'].iloc[i]),
+                        'model_name': model_name,
+                        'model_run_id': model_run_id,
+                        'model_deployment_date': model_deployment_date,
+                        'prediction_timestamp': str(datetime.now()),
+                        'rate_of_conversion': Decimal(str(raw_predictions['expectedConversionRate']['rateOfConversion'][i])),
```
Comment (jackblandin) [inline mlflow/pyfunc/scoring_server.py]: What is the `Decimal` class doing here? Is it just rounding?

Comment (brenoriba) [reply mlflow/pyfunc/scoring_server.py]: Can you also make sure we are uploading with 2 decimals?
Thanks!

Comment (PatrikSakac) [reply mlflow/pyfunc/scoring_server.py]: I used `Decimal` according to example in aws.
https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.Python.03.html
We are uploading with more than 2 decimals (for example 2051.6908658833427). Do we want just 2 decimals?

Comment (jackblandin) [reply mlflow/pyfunc/scoring_server.py]: Well, for rate of conversion, we will probably want more than two decimals. @brenoriba what was the reason for wanting only two decimals?

**Comment by jackblandin** [comment]
@PatrikSakac looks good. You can merge when you are ready.

**Pull request merged**